### DISCUSSION TITLE
I cannot access my account

### DISCUSSION THREAD

**Discussion by C0NN0RAD0** [Budibase/budibase#12665] [Help]
I have been able to access my budibase account for a while now, all of a sudden I cant use Microsoft account sign in nor can I reset my password. I Have errors when signing in with Microsoft saying I need to verify my SSO status and to check my email, so open this and it says email already verified, so I try to reset my password to which I have an error message saying unable to reset password, trying to sign up again says the email is already in use. Where do I turn? I cant contact support as I cannot log in and it's supposed to only be available to enterprise members, not surprised. And I cant message the Twitter page as it's for verified members only. It's not a very well thought out system in my opinion. 
<img width="621" alt="image" src="https://github.com/Budibase/budibase/assets/75914889/ffa10cab-e16d-4409-a71a-310a58c8826f">
<img width="319" alt="image" src="https://github.com/Budibase/budibase/assets/75914889/54e305ae-d06e-4e12-96db-865ac4b5c1c8">

**Comment by C0NN0RAD0** [comment]
The email address in question is Connor.caunt@students.plymouth.ac.uk

**Comment by ConorWebb96** [comment]
Closing as we are sorting this out via email, thanks @C0NN0RAD0.
